src/testdir/test_findfile.vim

```diff
@@ -479,6 +479,41 @@ func Test_findfunc_scriptlocal_func()
   call assert_equal('abc', g:FindFuncArg)
   bw!
 
+  new | only
+  set findfunc=
+  setlocal findfunc=NoSuchFunc
+  setglobal findfunc=s:FindFuncScript
+  call assert_equal('NoSuchFunc', &findfunc)
+  call assert_equal('NoSuchFunc', &l:findfunc)
+  call assert_equal(expand('<SID>') .. 'FindFuncScript', &g:findfunc)
+  new | only
+  call assert_equal(expand('<SID>') .. 'FindFuncScript', &findfunc)
+  call assert_equal(expand('<SID>') .. 'FindFuncScript', &g:findfunc)
```

Comment:
It seems that `findfunc'` is the only `'*func'` option for which `s:` is expanded to script ID. Maybe the expansion shouldn't be done?

Reply:
I think probably the other 'func' options should do this as well